libs/service-portal/settings/access-control/src/screens/Access/Access.tsx

```diff
@@ -21,6 +59,76 @@ function Access() {
             'Reyndu að lámarka þau réttindi sem þú vilt veita viðkomandi eins mikið og mögulegt er.',
         })}
       />
+      <FormProvider {...hookFormData}>
+        <form onSubmit={onSubmit}>
+          <Box marginBottom={3} display="flex" justifyContent="flexEnd">
+            <Inline space={3}>
+              <Button
+                onClick={() => console.log('delete')}
+                variant="ghost"
+                colorScheme="destructive"
+                size="small"
+                icon="close"
+              >
+                {formatMessage({
+                  id: 'service.portal:access-control-access-remove',
+                  defaultMessage: 'Eyða umboði',
+                })}
+              </Button>
+              <Button size="small" type="submit" icon="checkmark">
+                {formatMessage({
+                  id: 'service.portal:access-control-access-save',
+                  defaultMessage: 'Vista aðgang',
+                })}
+              </Button>
+            </Inline>
+          </Box>
+
+          <T.Table>
+            <T.Head>
+              <T.Row>
+                <T.HeadData>
+                  <Text variant="small" color="blue600">
+                    {formatMessage({
+                      id: 'service.portal:access-control-access-access',
+                      defaultMessage: 'Aðgangur',
+                    })}
+                  </Text>
+                </T.HeadData>
+                <T.HeadData>
+                  <Text variant="small" color="blue600">
+                    {formatMessage({
+                      id: 'service.portal:access-control-access-explanation',
+                      defaultMessage: 'Útskýring',
+                    })}
+                  </Text>
+                </T.HeadData>
+                <T.HeadData>
+                  <Text variant="small" color="blue600">
+                    {formatMessage({
+                      id: 'service.portal:access-control-access-created',
+                      defaultMessage: 'Í gildi frá',
+                    })}
```

Comment:
These should be namespaced separately, eg under service portal settings or access-control namespaces.